### PR TITLE
updated the contributing link of readme

### PR DESCRIPTION
**Pull request by blulady**
Changed contributing link in the readme (not coc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ for issue in Issue.get_content_issues('issue_header'):
 
 # Contributing
 
-Review the [Contributing Guidelines](CONTRIBUTING.md) for information on how to contribute to this project.
+Review the [Contributing Guidelines](https://github.com/Python-Community-News/GH-Issues/blob/main/CONTRIBUTING) for information on how to contribute to this project.
```
Comment (kjaymiller) [inline README.md]: The Contributing Guidelines are shared amongst the org file

```suggestion
Review the [Contributing Guidelines](https://github.com/Python-Community-News/.github/blob/main/CONTRIBUTING) for information on how to contribute to this project.
```

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Just need to change the link

File: README.md (hunk 1 above)
Comment (blulady) [reply README.md]: I have no idea how I did that... I am almost impressed with how many times I have ruined this PR. Thanks for your patience.

**Comment by kjaymiller** [comment]
We're actually going to be standardizing on the README in a different way. Thank you for your work on this. I hope you'll contribute to other parts of the code-base

**Pull request closed without merging**